src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -1027,7 +1034,17 @@ export const modelingMachineCommandConfig: StateMachineCommandSetConfig<
       nodeToEdit: {
         ...nodeToEditProps,
       },
+      objects: {
+        // selectionMixed allows for feature tree selection of module imports
+        inputType: 'selectionMixed',
+        selectionTypes: ['path', 'sweep', 'compositeSolid'],
+        selectionFilter: ['object'],
+        multiple: true,
+        required: true,
+        hidden: (context) => Boolean(context.argumentsToSubmit.nodeToEdit),
+      },
       color: {
+        // TODO: change this to a color picker
```

Comment:
This list of options is dumb, but I didn't want to make the color picker input as part of this PR considering how huge it is already. Will make a seperate issue for it if that's cool!

Reply:
absolutely.

Reply:
Awesome! Created at https://github.com/KittyCAD/modeling-app/issues/7811